webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/core_menu_options.test.tsx

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {DateTime} from 'luxon';
+import React from 'react';
+
+import {renderWithContext, fireEvent, screen} from 'tests/react_testing_utils';
+
+import CoreMenuOptions from './core_menu_options';
+
+jest.mock('components/advanced_text_editor/use_post_box_indicator', () => ({
+    __esModule: true,
+    default: jest.fn(),
+}));
+
+jest.mock('components/menu', () => ({
+    Item: jest.fn(({labels, trailingElements, children, ...props}) => (
+        <div {...props}>
+            {labels}
+            {children}
+            {trailingElements}
+        </div>
+    )),
+}));
+
+jest.mock('components/timestamp', () => jest.fn(({value}) => <span>{value}</span>));
```

Comment:
Does this need to be mocked? When I was testing my other suggestions, this seemed to work fine without it mocked